### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by OrangeJedi** [OrangeJedi/Aerial#118] [Announcements]
## This new release includes the following new features:
* Update notifier
    * Aerial will now send a push notification on startup if a new release is available
    * A notification badge with a link to the GitHub releases page will also show in the config menu
* A new (long-overdue) `About` tab 
    * Contains about, versioning, and licensing information 
* Automatic sunrise & sunset times based on GPS coordinates (see #61)
    * This setting can be found under the new `Time & Location` tab in the menu. It must be set manually  
* A new text type, `Astronomy`
    * Display the sunrise/sunset or moonrise/moonset time on your screensaver! 
* Enhanced suspend features
    * Use the tray menu to temporarly suspend Aerial for 1 or 3 hours. Useful for when you don't need your gorgeous screensaver to run every few minutes. (See #114)

### It also fixes the following bugs:
* Misc development issues
* #110 - video not playing on multiscreen. A big thanks to @Alexyoe for finding and fixing this bug!
    * If you were having troubles, check the `Alternate Render Method` setting under the `Multiscreen` menu
    * This fix also adds a custom FPS setting when enabled
* #116 - cannot read undefined error
* Multiple repairs related to custom videos

## How to Install
Download `Aerial Setup 1.1.0.exe` (or download the .zip and extract it) and run it. Aerial takes care of the rest. A configuration window will open where you can adjust the settings.

🚨Disable any other screensavers before installing and make sure to set your computer sleep settings to longer than the screensaver start time.

<hr /><em>This discussion was created from the release <a href='https://github.com/OrangeJedi/Aerial/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by yachicha83** [comment]
Hello!
Thanks for your product! I've been using it for many years now.
Request. The screensaver turns on while watching a movie in the player or a video on YouTube. Can you fix this?
Thank you!

**Reply by OrangeJedi** [reply]
Hi, I'm glad that you enjoy Aerial. We've noticed this problem and are discussing it in #114. 

The TL;DR is that there isn't a super easy way to fix this, but we are looking at a few solutions.

**Comment by rjarjour** [comment]
Dear @OrangeJedi 

Thank you for considering the Windows Team! I am wondering where and how I download the videos or do they automatically download and update? 
I have already installed "_aerial Setup 1.1.0.exe"_ and got the "_aerial config_".  What others steps should I do to make the screensaver work on the windows PC? The video cache is 0MB
![Untitled](https://user-images.githubusercontent.com/122150646/211143461-c926a99e-a95e-4354-9128-6bb579c79e57.jpg)

**Reply by OrangeJedi** [reply]
It looks like you've taken the right steps to have the video download automatically. Because of how long the process can take Aerial only downloads videos when running in screensaver mode. After the screensaver runs for a few minutes (depending on your internet speed) you should see files in the downloads cache. Let me know if there are still problems after a few days.

**Comment by rjarjour** [comment]
Thank you for your response first forever after downloading it. It kept showing me an error message, and saying that JavaScript is error. Also, the touchpad stopped working. I don’t know if the issue is from the software stuff or my PC is not compatible. I tried to close the aerial configuration software and reopen it again but keeps showing error message. Leslie, the screensaver on my PC, is set to “none” as it does not show the aerial screensaver option.
But anyway, I will wait for a few days and see how it goes. Thank you so much :-)

Kind regards,
Rami Jarjour

On 7 Jan 2023, at 7:01 pm, OrangeJedi ***@***.***> wrote:

﻿

It looks like you've taken the right steps to have the video download automatically. Because of how long the process can take Aerial only downloads videos when running in screensaver mode. After the screensaver runs for a few minutes (depending on your internet speed) you should see files in the downloads cache. Let me know if there are still problems after a few days.

**Comment by modestin** [comment]
Hi there!! I love this tool! Its the best one in my opinion, in comparison to other tools that now are older, but that did the same. 

Speaking of which... I noticed some videos from my other versions, are not present here in yours... so what I did was to identify those and move it to a separate folder, to use them as custom videos.

Unfortunatelly, it seems that the custom videos selection here is broken :-( ... whenever I try to select the folder that I want to use as the one for custom videos, after clicking "Add Folder", and then "Select Folder"  nothing happens. 

Am I doing something wrong? or is it actually something broken here? ... thanks in advance!

![image](https://user-images.githubusercontent.com/30641663/211186502-ec951acc-3cd8-419a-9855-0dbe94c8a69b.png)
